scalafix/project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 resolvers += Resolver.sonatypeRepo("releases")
 
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.10.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.10.1")
```

Comment:
Oh, woops, I take that back. I guess we need it for this nested project, unless we replace this with `sbt-http4s-org` too?